src/main/java/com/sprint/mission/discodeit/controller/BinaryContentController.java

```diff
@@ -0,0 +1,38 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.entity.BinaryContent;
+import com.sprint.mission.discodeit.service.BinaryContentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RequiredArgsConstructor
+@Controller
+@ResponseBody
+@RequestMapping("/api/binaryContents")
```

Comment:
보통 url은 카멜케이스보다는 `-`(하이픈)을 주로 사용해요.   
```suggestion
@RequestMapping("/api/binary-contents")
```

Reply:
넵! 말씀해주신 부분 수정하도록하겠습니다 감사합니다 !